lib/winapi/basetyps.h

```diff
@@ -0,0 +1,5 @@
+#include <guiddef.h>
+
+typedef GUID IID;
+typedef IID *REFIID;
```

Comment:
These types never seem to be used in this PR.
What will they be used for?

I mostly know these from COM - and we likely won't implement such features in nxdk.

Reply:
These typedefs are required by SDL's threading implementation which I used to test this code.

Reply:
The only one we might include is <nxdk-sdl>/src/core/windows/SDL_windows.h which also drags in a lot of other Windows garbage (\- COM related; `CoInitializeEx` etc.). How is that dealt with in nxdk?

It feels like we should disable those parts in SDL for Xbox compilation.

Or am I looking at wrong code? Where do these appear?

Reply:
`SDL_windows.h` is indeed the file where this is required. We currently never build any code that uses the stuff declared there, that's why we don't get any unresolved references or have to deal with `CoInitialize`-stuff.

It's probably better to work around this in SDL instead - if you agree that I should remove it here I'll probably do that later today.

Reply:
Yes, we should remove this stuff from SDL, when compiling for Xbox

Reply:
Done.

Reply:
Clarification: "Done" by removing these types.
No nxdk-sdl changes have been shown yet.